pkg/print/config.go

```diff
@@ -0,0 +1,23 @@
+package print
+
+import (
+	"github.com/fatih/color"
+	configtypes "github.com/replicatedhq/kots/pkg/kotsadmconfig/types"
+	"github.com/replicatedhq/kots/pkg/logger"
+)
+
+func ConfigValidationErrors(log *logger.CLILogger, groupValidationErrors []configtypes.ConfigGroupValidationError) {
+	errPrint := color.New(color.FgHiRed)
+	errPrint.Println("Following config items have validation errors:")
```

Comment:
why not use the `log` parameter that's passed? you can build the string and the print it once using the logger. also, i'd suggest adding a new line at the beginning and end for better readability.